Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -18,7 +18,7 @@
 #include <string>
 
 enum struct PhysicalSpecies{unspecified=0, electron, positron, photon, hydrogen, helium, boron,
```

Comment:
Let's update the docs as well at: https://warpx.readthedocs.io/en/latest/usage/parameters.html?highlight=copper#particle-initialization